Codigo/src/file-coverage/models/file-coverage.ts

```diff
@@ -34,12 +35,43 @@ export class FileCoverage {
       textEditor,
       this.lcovFiles
     );
-
+    
     const coverageLines = this.lcovFilesToCoverageLines(lcovFiles);
+
     if (coverageLines) {
       return coverageLines;
     }
 
+    const isFileDiff = await this.gitService.getIsCurrentFilesBranchDiff(
+      textEditor.document.fileName
+    );
+
+    if (isFileDiff) {
+      const branchDiff = await this.gitService.getCurrentBranchDiff(
+        textEditor.document.fileName
+      );
+
+      const filteredFull = coverageLines?.full.filter((line) => {

```

Comment:
depois de trocar a condicional do comentario de cima pode remover esse operador:
```suggestion
      const filteredFull = coverageLines.full.filter((line) => {

```